### ISSUE TITLE
Comments in C++ have trailing semicolon

### ISSUE THREAD

**Issue by jayvdb** [adsharma/py2many#384]
Using `.comment(..)` with `._statement_separator` of `;` results in
```cpp
/* foo */;
```
which triggers cpplint
> tests/expected/comment_unsupported.cpp:7:  Line contains only semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]

**Comment by adsharma** [comment]
ok to close?

**Comment by jayvdb** [comment]
No.  I have just ignored the check type.  This generating problem still occurs.  A slightly better situation would be to add a per-line rule disable on the problematic lines, but we really shouldnt leave them in the source.  It looked non-trivial to fix during generation, so might be easier to fix with post-processing.  I wouldnt be surprised if we can find a linter/formatter which can remove them.  If clang-format & clang-tidy, cant fix this, we should raise an enhancement request there.
